sentry-android-core/src/main/java/io/sentry/android/core/internal/modules/AssetsModulesLoader.java

```diff
@@ -0,0 +1,44 @@
+package io.sentry.android.core.internal.modules;
+
+import android.content.Context;
+import io.sentry.ILogger;
+import io.sentry.SentryLevel;
+import io.sentry.internal.modules.ModulesLoader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.ref.WeakReference;
+import java.util.Map;
+import java.util.TreeMap;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public final class AssetsModulesLoader extends ModulesLoader {
```

Comment:
L: Method `getProguardUUID` also uses `context.getAssets()`, maybe we move that method to this modules loader as well?
Maybe we rename the interface to something more generic, like AssetsLoader instead? having methods for the modules and proguard.

Reply:
hmm, not sure, the only reason I created this class is because plain java also needs a ModulesLoader, hence different implementations. I think there's also not much in common for proguarduuid and modules besides opening a stream and catching a few exceptions.

If there's a new thing which we have to read from the assets I'll think about how to better generalize it, but for now I'd just keep it like this

Reply:
Makes sense, your call to refactor this later if needed.